testing/databaseTest.py

```diff
@@ -395,8 +395,9 @@ def test_metal_libraries(self):
     # These are the actual tests, that don't start with a "test_" name:
     def kinetics_check_surface_training_reactions_can_be_used(self, family_name):
         """Test that surface training reactions can be averaged and used for generating rate rules"""
-        family = self.database.kinetics.families[family_name]
-        family.add_rules_from_training(thermo_database=self.database.thermo)
+            family = self.database.kinetics.families[family_name]
+            if not family.auto_generated:
+                family.add_rules_from_training(thermo_database=self.database.thermo)
```

Comment:
Is there a reason we don't use `family.fill_rules_by_averaging_up(verbose=True)` for this surface kinetics unit test?

Reply:
To be honest, I don't know why this isn't added. This test was written by a graduated student from the Northeastern Group. I personally feel there is no harm in testing the function you mentioned as well, so I just added it in a new commit. (I also quickly checked KineticsTest.py, and it seems to me there isn't a duplicate test for surface families there).

Reply:
Sounds good thanks